tests/testthat/_snaps/map.md

```diff
@@ -22,3 +22,25 @@
       Error in `purrr::map_int()`:
       ! Result 2 must have length 1, not 2.
 
+# requires output be length 1 and have common type
+
+    Code
+      map_vec(1:2, ~ rep(1, .x))
+    Condition
+      Error in `map_vec()`:
+      ! All elements must be size 1.
+      i `out[[2]]` is size 2.
```

Comment:
To confirm, you want the `error_arg` to be `out`?

Reply:
Yup